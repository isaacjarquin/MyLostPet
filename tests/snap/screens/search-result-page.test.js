import 'react-native'
import React from 'react'
import SearchResultPage from '../../../screens/search-result-page'

import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const pet = {name: 'name'}
  const navigation = {
    state: {
      params: {
        pets: [pet]
      }
    }
  }

  const tree = renderer.create(
    <SearchResultPage navigation={navigation} />
    ).toJSON()
  expect(tree).toMatchSnapshot()
})
