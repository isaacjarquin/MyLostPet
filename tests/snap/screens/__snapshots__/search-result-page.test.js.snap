// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <RCTScrollView
    dataSource={
      ListViewDataSource {
        "items": 1,
      }
    }
    renderRow={[Function]}
    renderScrollComponent={[Function]}
  >
    <View>
      <View
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#fff",
            "flex": 1,
            "justifyContent": "flex-start",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "backgroundColor": "white",
                "borderColor": "#e1e8ee",
                "borderWidth": 1,
                "margin": 15,
                "marginBottom": 0,
                "padding": 15,
                "shadowColor": "rgba(0,0,0, .2)",
                "shadowOffset": Object {
                  "height": 0,
                  "width": 0,
                },
                "shadowOpacity": 1,
                "shadowRadius": 1,
              },
              Object {
                "padding": 0,
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                },
                undefined,
                undefined,
              ]
            }
          >
            <View
              style={undefined}
            >
              <View
                style={
                  Array [
                    Object {
                      "height": 150,
                      "width": null,
                    },
                    undefined,
                  ]
                }
              >
                <Image
                  resizeMode="cover"
                  source={
                    Object {
                      "uri": undefined,
                    }
                  }
                  style={
                    Array [
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      },
                      Object {
                        "height": undefined,
                        "width": undefined,
                      },
                      undefined,
                    ]
                  }
                />
                <View
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "alignSelf": "stretch",
                        "backgroundColor": "rgba(0, 0, 0, 0.2)",
                        "bottom": 0,
                        "flex": 1,
                        "justifyContent": "center",
                        "left": 0,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    disabled={false}
                    ellipsizeMode="tail"
                    style={
                      Array [
                        Object {},
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        Array [
                          Object {
                            "color": "white",
                            "fontSize": 22.5,
                            "fontWeight": "800",
                            "marginBottom": 8,
                          },
                          undefined,
                        ],
                      ]
                    }
                  />
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    disabled={false}
                    ellipsizeMode="tail"
                    style={
                      Array [
                        Object {},
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        Array [
                          Object {
                            "color": "white",
                            "fontSize": 16.25,
                            "fontWeight": "400",
                            "marginBottom": 8,
                          },
                          undefined,
                        ],
                      ]
                    }
                  />
                </View>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "padding": 10,
                    },
                    undefined,
                  ]
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "fontWeight": "bold",
                      "marginBottom": 10,
                    }
                  }
                >
                  Encontrado en 
                  , el 
                </Text>
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "marginBottom": 20,
                    }
                  }
                />
                <View
                  style={
                    Array [
                      Object {
                        "backgroundColor": "transparent",
                        "marginLeft": 15,
                        "marginRight": 15,
                      },
                      undefined,
                      undefined,
                      undefined,
                    ]
                  }
                >
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Array [
                        Object {
                          "backgroundColor": "transparent",
                        },
                        undefined,
                      ]
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "#9E9E9E",
                            "flexDirection": "row",
                            "justifyContent": "center",
                            "padding": 19,
                          },
                          undefined,
                          undefined,
                          undefined,
                          undefined,
                          undefined,
                          undefined,
                          undefined,
                          Object {
                            "padding": 12,
                          },
                          undefined,
                          undefined,
                          undefined,
                          Object {
                            "borderRadius": 0,
                            "marginBottom": 0,
                            "marginLeft": 0,
                            "marginRight": 0,
                          },
                          undefined,
                          undefined,
                        ]
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        disabled={false}
                        ellipsizeMode="tail"
                        style={
                          Array [
                            Object {},
                            undefined,
                            undefined,
                            undefined,
                            undefined,
                            undefined,
                            undefined,
                            undefined,
                            undefined,
                            undefined,
                            Array [
                              Object {
                                "color": "white",
                                "fontSize": 20,
                              },
                              undefined,
                              Object {
                                "fontSize": 17.5,
                              },
                              undefined,
                              undefined,
                              undefined,
                              undefined,
                            ],
                          ]
                        }
                      >
                        Contactar
                      </Text>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
